bundles/org.openhab.binding.tibber/README.md

```diff
@@ -102,6 +102,7 @@ Tibber API will be auto discovered if provided input is correct.
 
 ## Tomorrow and Today Prices
 
+The today and tomorrow prices are served as time series on the _Current Total_ channel and as json data on the channels _Today prices_ and _Tomorrow prices_. 
```

Comment:
Perhaps it would be better to mention the concrete/actual names of the channels?
```suggestion
The today and tomorrow prices are served as time series on the `current_total` channel and as JSON data on the channels `today_prices` and `tomorrow_prices`.
```

Reply:
I've just taken the value via C&P from the table above. So remarked, that they're all wrong and corrected that.
Added a column "forecast" so it's clear when mapping the channels and no need to read all the text below :)